pycontw/pycontw-frontend
# fix : 2021 轉換靜態頁

## Types of changes

build(nuxt-generate): add dynamic routes configuration and update to use asyncData fetching data

## Description


## Steps to Test This Pull Request


1. npm run json-server
2. npm run generate
3. npm run start
4. Check out if all pages works well.

## Expected behavior

All pages should show up data as usual.
## Related Issue

fix #338 

## Additional context
